services/core/shared/core/compat/sdk_v5/transactions.js

```diff
@@ -72,42 +77,55 @@ const normalizeTransaction = tx => {
 	return tx;
 };
 
-const getTransactions = async params => {
-	const transactionsDB = await knex('transactions');
-	const transactions = {
-		data: [],
-		meta: {},
-	};
+const validateParams = async params => {
 	if (params.fromTimestamp || params.toTimestamp) {
 		params.propBetween = {
 			property: 'timestamp',
 			from: Number(params.fromTimestamp) || 0,
 			to: Number(params.toTimestamp) || Math.floor(Date.now() / 1000),
 		};
 	}
+	if (params.sort && params.sort.includes('nonce')) {
+		if (!params.senderId) return new Error('Nonce based sorting is only possible along with senderId');
+	}
```

Comment:
These conditions can be merged.
```suggestion
	if (params.sort && params.sort.includes('nonce') && !params.senderId) {
		return new Error('Nonce based sorting is only possible along with senderId');
	}
```